sass/components/_asset-card.scss

```diff
@@ -106,4 +107,25 @@ $asset-card-padding: 0.4rem;
             background-color: rgb(110, 110, 247);
         }
     }
+}
+
+@media (min-width: 285px) {
+    .asset-card{
+        max-width: 100vw !important;
+    }
+}
+@media (min-width: 315px) {
+    .asset-card{
+        aspect-ratio: auto 10/6 !important;

```

Comment:
Can we extract this into a variable and use the same value for every breakpoint?

Reply:
Ah, I see, this is the aspect ratio of the entire card rather than just the banner area.